libtiff/libtiff_ctypes.py

```diff
@@ -903,6 +901,37 @@ def SetDirectory(self, dirnum):
     setdirectory = SetDirectory
 
     @debug
+    def SetSubDirectory(self, diroff):
+        """
+        Changes the current directory
+        and reads its contents with TIFFReadDirectory.
+        The parameter dirnum specifies the subfile/directory as
+        an integer number, with the first directory numbered zero.
+
+        SetSubDirectory acts like SetDirectory,
+        except the directory is specified as a file offset instead of an index;
+        this is required for accessing subdirectories
+        linked through a SubIFD tag.
+
+        Parameters
+        ----------
+        diroff: int
+            The offset of the subimage. It's important to notice that
+            it is not an index, like dirnum on SetDirectory
+
+        Returns
+        -------
+        int
+            On successful return 1 is returned.
+            Otherwise, 0 is returned if dirnum or diroff
+            specifies a non-existent directory,
+            or if an error was encountered
+            while reading the directory's contents.
+        """
+        return libtiff.TIFFSetSubDirectory(self, diroff)
+    setsubdirectory = SetSubDirectory
```

Comment:
No need to create also a lowercase version of that function. The only reason it is like this is for compatibility with some older versions of pylibtiff which had all wrappers named in lowercase.